src/test/java/com/microsoft/sqlserver/jdbc/callablestatement/CallableStatementTest.java

```diff
@@ -99,11 +99,11 @@ public static void setupTest() throws Exception {
             TestUtils.dropProcedureIfExists(conditionalSproc, stmt);
             TestUtils.dropProcedureIfExists(simpleRetValSproc, stmt);
             TestUtils.dropProcedureIfExists(zeroParamSproc, stmt);
-            TestUtils.dropUserDefinedTypeIfExists(manyParamUserDefinedType, stmt);
             TestUtils.dropProcedureIfExists(manyParamProc, stmt);
+            TestUtils.dropProcedureIfExists(procedureNameJSON, stmt);
             TestUtils.dropTableIfExists(manyParamsTable, stmt);
             TestUtils.dropTableIfExists(tableNameJSON, stmt);
-			TestUtils.dropProcedureIfExists(procedureNameJSON, stmt);
+            TestUtils.dropUserDefinedTypeIfExists(manyParamUserDefinedType, stmt);
```

Comment:
Tens of thousands of stray types were not getting cleaned up due to this ordering. The type was referenced by the sproc, so the type wouldn't get dropped.